README.md

```diff
@@ -4,10 +4,31 @@
 
 This project is experimental and not yet ready for production use.
 
-
-# How To Use
-
-TODO
+## Supported Targets
+
+- ESP8266
+- ESP32
+- ESP32-S2
+- ESP32-S3
+- ESP32-C2
+- ESP32-C3
+- ESP32-C5
+- ESP32-C6
+- ESP32-C61
+- ESP32-H2
+- ESP32-P4
```

Comment:
We will need to add beta chips here as well time-to-time. Will that be a problem for you?

Reply:
No. I don't think we will have an issue as long as we keep the common structure.